src/ResumeApplicationToast.js

```
@@ -0,0 +1,110 @@
+    // Show offer toast and hide mail toast
+    if (toastMail) toastMail.style.display = "none";
+    toastOffer.style.display = "flex";
+
+    // Update link destination for the offer toast link
+    const offerLink = toastOffer.querySelector("a");
+    if (offerLink) {
+      offerLink.addEventListener("click", async (e) => {
+        e.preventDefault();
+        try {
+          const response = await ResumeApplicationToast.fetchRedirectLink();
+          if (response && response.url) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)